includes/class-content-toc.php

```diff
@@ -233,8 +233,8 @@ public function filter_content( $post_content, $shortcode_atts ) {
 			$list_html = sprintf(
 				'<%1$s%2$s>%3$s</%1$s>',
 				esc_attr( $this->settings['list_tag'] ),
-				$this->tag_class( $this->settings['list_class'] ),
-				$items_html
+				esc_attr( $this->tag_class( $this->settings['list_class'] ) ),
```

Comment:
I would alter `tag_class()` so it just returns the attribute value, and then use `class="%2$s"` in your formatted strings.